spec/flipper/api/v1/actions/actors_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'helper'
+
+RSpec.describe Flipper::Api::V1::Actions::Actors do
+  let(:app) { build_api(flipper) }
+  let(:actor) { Flipper::Actor.new('User123') }
+
+  describe 'GET /actors/:flipper_id' do
+    before do
+      flipper[:my_feature_1].enable
+      flipper[:my_feature_2].disable
+      flipper[:my_feature_3].enable_actor(actor)
+    end
+
+    context 'when no feature is specified' do
+      before do
+        get "/actors/#{actor.flipper_id}"
+      end
+
+      it 'responds with success' do
+        expect(last_response.status).to eq(200)
+      end
+
+      it 'returns all features' do
+        expected_response = {
+          'flipper_id' => 'User123',
+          'features' => [
```

Comment:
What are your thoughts on making this `'features' => {'my_feature_1' => {'enabled' => true}}`. The Hash lookup would make it easier to get at an individual feature. Keeping 'enabled' => true as a hash means we can always add more keys/values if we need. Also, 'features' being a hash means we could always add pagination information there if we ever paginate features due to the sheer number people could have.

Reply:
Yep, good call 👍